### PR TITLE
Removed extraneous travis configs, test against latest node

### PR DESCRIPTION
**Pull request by amilajack**
Remove travis support for legacy node

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,6 @@ branches:
 before_script:
 - npm install -g gulp
 - gulp
-script: npm test
```
Comment (RunDevelopment) [inline .travis.yml]: Why did you remove the testing part itself ?? 😕

Comment (amilajack) [reply .travis.yml]: Travis defaults to this. No need to include. But if you want me to revert this I can

File: .travis.yml [hunk 2]
```diff
@@ -12,7 +12,8 @@ branches:
...
+test:
+  - npm test
```
Comment (mAAdhaTTah) [inline .travis.yml]: This should remain `script: npm test`. `test` is not a valid configuration for Travis.

**Comment by RunDevelopment** [comment]
Closed because #1651 was merged,
